Build fails building FStream.cpp.o in OS X 10.9.2

After running git pull and git submodule update this morning, then removing my old nta dir and running cmake and then make per the README.md I get this error:

```
[ 1%] Building CXX object CMakeFiles/os.dir/nta/os/FStream.cpp.o
/usr/bin/c++ -mmacosx-version-min=10.7 -std=c++98 -I/Users/iandanforth/nupic -isystem/Users/iandanforth/nupic/external/common/include -isystem/Users/iandanforth/nupic/external/darwin64/include -fPIC -DPIC -m64 -DHAVE_CONFIG_H -DNTA_INTERNAL -DNTA_PLATFORM_darwin64 -DBOOST_NO_WREGEX -DNUPIC2 -fvisibility=hidden -Wall -Wreturn-type -Wunused -Wno-unused-parameter -gfull -DNTA_ASSERTIONS_ON -DNTA_PYTHON_SUPPORT=2.7 -isystem/Library/Python/2.7/site-packages/numpy/core/include -isystem/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -O3 -pipe -DNTA_ASM -o CMakeFiles/os.dir/nta/os/FStream.cpp.o -c /Users/iandanforth/nupic/nta/os/FStream.cpp
/Users/iandanforth/nupic/nta/os/FStream.cpp:186:19: error: cannot initialize a variable of type 'char ' with an rvalue of type 'int'
char retval = ::strerror_r(error, errbuf, 256);
```

--- a/src/main/os/FStream.cpp
+++ b/src/main/os/FStream.cpp
@@ -181,9 +181,7 @@ void *ZLib::fopen(const std::string &filename, const std::string &mode,
           case Z_MEM_ERROR: message = "Zlib memory error."; break;
           case Z_BUF_ERROR: message = "Zlib buffer error."; break;
           case Z_VERSION_ERROR: message = "Zlib version error."; break;
-          default:
-            message = ::strerror(error);
-            break;
+          default: message = ::strerror(error); break;
         }   
         if(errorMessage) {
           *errorMessage = message;

